package.json

```
@@ -166,7 +166,7 @@
     "playwright": "^1.50.1",
     "postcss": "^8.4.49",
     "postcss-lit": "^1.1.1",
-    "prettier": "3.5.1",
+    "prettier": "3.5.2",
```

Do we know why this one's pinned?  Couldn't it be?:

```suggestion
    "prettier": "^3.5.2",
```

---

@clintcs do you happen to recall? I don't!

---

Looks like it's been pinned since the beginning. Probably shouldn't be unless we decide to pin everything as a follow-up.

---

I'm good with `^`